Source/WebCore/PAL/pal/system/mac/PopupMenu.mm

```diff
@@ -53,20 +53,30 @@ void popUpMenu(NSMenu *menu, NSPoint location, float width, NSView *view, int se
     }
 
     // These numbers were extracted from visual inspection as the menu animates shut.
-    NSSize labelOffset = NSMakeSize(11, 1);
+    NSSize labelOffset;
     if (menu.userInterfaceLayoutDirection == NSUserInterfaceLayoutDirectionRightToLeft)
         labelOffset = NSMakeSize(24, 1);
+    else
```

Comment:
This seems strictly worse.

Reply:
I think avoiding an extra construction is worth it, unless the compiler knows how to optimize that out as it was before.